src/code/z_message.c

```diff
@@ -576,9 +5035,825 @@ void Message_Draw(PlayState* play) {
     CLOSE_DISPS(gfxCtx);
 }
 
+s16 sTextboxXPositions[] = {
+    0x22, 0x22, 0x22, 0x22, 0x22, 0x22, 0x22, 0x22, 0x22, 0x22, 0x22, 0x22, 0x22, 0x22, 0x22, 0x22,
+};
+s16 sTextboxLowerYPositions[] = {
+    0x8E, 0x8E, 0x8E, 0x8E, 0xAE, 0x8E, 0x8E, 0x8E, 0x8E, 0x82, 0xAE, 0, 0x8E, 0x8E, 0x8E, 0x8E,
+};
+s16 sTextboxUpperYPositions[] = {
+    0x26, 0x26, 0x26, 0x26, 0xAE, 0x26, 0x26, 0x26, 0x26, 0x3C, 0xAE, 0, 0x26, 0x26, 0x26, 0x26,
+};
+s16 sTextboxMidYPositions[] = {
+    0x5A, 0x5A, 0x5A, 0x5A, 0xAE, 0x5A, 0x5A, 0x5A, 0x5A, 0x5A, 0xAE, 0, 0x5A, 0x5A, 0x5A, 0x5A,
+};
```

Comment:
Decimal? Also all seem to be indexing `TEXTBOX_TYPE_X`, so `TEXTBOX_TYPE_MAX` plus comments?